client/components/credit-card-form-fields/index.jsx

```diff
@@ -26,28 +28,47 @@ class CreditCardFormFields extends React.Component {
 		onFieldChange: PropTypes.func.isRequired,
 	};
 
-	field = ( fieldName, componentClass, props ) => {
+	constructor( props ) {
+		super( props );
+		this.state = {
+			countryCode: '',
+			phoneCountryCode: '',
+		};
+	}
+
+	shouldComponentUpdate( nextProps ) {
+		if ( ! isEqual( nextProps.card, this.props.card ) ) {
+			return true;
+		}
+		return false;
+	}
+
+	createField = ( fieldName, componentClass, props ) => {
 		return React.createElement(
 			componentClass,
-			assign( {}, props, {
-				additionalClasses: 'credit-card-form-fields__field',
-				eventFormName: this.props.eventFormName,
-				isError: this.props.isFieldInvalid( fieldName ),
-				name: fieldName,
-				onBlur: this.handleFieldChange,
-				onChange: this.handleFieldChange,
-				value: this.getFieldValue( fieldName ),
-				autoComplete: 'off',
-			} )
+			Object.assign(
+				{},
+				{
+					additionalClasses: 'credit-card-form-fields__field',
+					eventFormName: this.props.eventFormName,
+					isError: this.props.isFieldInvalid( fieldName ),
+					name: fieldName,
+					// onBlur: this.handleFieldChange,
```

Comment:
Why is this comment out?

Reply:
Comment was checked in by accident :)

I couldn't find a use case for the `onBlur` event handler, which fires the same callback as `onChange` anyway. 

`onChange` already updates the model, and we're not doing any real-time error checking, so it appeared redundant. Unless I'm missing something?